ofx.go

```diff
@@ -46,11 +47,16 @@ const (
 )
 
 type Amount struct {
-	value big.Rat
```

Comment:
true... I changed code to just export value, so it can be used from external applciations